app/StatusModule/WebHook.js

```diff
@@ -0,0 +1,49 @@
+var StatusModule = require('./StatusModule');
+
+/**
+ * WebHook
+ *
+ * @param {object} config
+ * @param {StatusManager} statusManager
+ * @constructor
+ */
+var WebHook = function(config, statusManager) {
+    StatusModule.call(this, config, statusManager);
+};
+
+/**
+ * Sets the http module.
+ */
+WebHook.prototype.init = function() {
+    this.https = require('https');
+};
+
+/**
+ * Handle the incoming statuses
+ */
+WebHook.prototype.handleStatus = function() {
+    var webHook,
+        request;
```

Comment:
Please use a var per variable (linter should catch this, but I guess I didn't set it up properly :wink:)